docs/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Next
 
 * Items in the Postmaster which cannot be pulled are shown faded out.
+* Search filters that operate on power levels now accept the keywords "hardcap" and "softcap" to refer to the current season's power limits. e.g "power:>=softcap"
```

Comment:
there's 3 caps:
soft cap (new drops are always more powerful until this number)
hard cap/powerful cap (powerful drops are always more powerful until this number)
and
hard cap/pinnacle cap (absolute max)
this season for instance, they're 1200, 1250 and 1260

two things are referred to as "hard cap" depending on who you are talking to, so i am left with 2 questions:
"shouldn't there be three keywords"
but also
"is there any situation powerful cap or soft cap would _really_ need to be measured for?" especially soft cap. you can't stop progressing at it, the game naturally will dump powerfuls on you. it's not a meaningful distinction at all for any purpose i can think of.

Reply:
Hmm, the intention was for this to be used in folks' "what to keep" searches, and for me a lot of times that's stuff over the powerful cap since it tends to be more useful for infusing stuff.

Reply:
really only turns out useful by coincidence if your PL happens to be near softcap, but i guess this does satisfy my demand for an example usage :)

Reply:
Mostly I'm trying to help out this gargantuan filter from @TMMania by allowing the "hardcap" alias: https://github.com/TMMania/TMManias-DIM-Filter-Gallery/blob/main/Master%20Filter#L80

"softcap" is something I'd probably use since I can usually get to the "powerful cap" in a season but rarely actually hit the hard cap.

I'm super open to better words for these though.

Reply:
My two bits on naming: I'm a fan of both (softcap, hardcap) and (powerfulcap, pinnaclecap) but thinking about it now, powerfulcap and pinnaclecap works better since it follows a similar nomenclature as D2 which would reduce confusion on the function when you're not looking at the function description.